vscode-extension/package.json

```diff
@@ -112,7 +112,17 @@
           "description": "Whether or not we should automatically start the Relay Compiler in watch mode when you open a project."
         }
       }
-    }
+    },
+    "jsonValidation": [
+      {
+        "fileMatch": "relay.config.json",
+        "url": "relay://relay-config-schema"
```

Comment:
TIL You can define your own url schemes. This is perfect. I wasn't sure how we were going to be able to locate the user's local copy of the Json Schema, but this looks like a great approach.